plugins/rtmp-services/rtmp-services-main.c

```diff
@@ -18,8 +17,8 @@ MODULE_EXPORT const char *obs_module_description(void)
 	return "OBS core RTMP services";
 }
 
-#define RTMP_SERVICES_LOG_STR "[rtmp-services plugin] "
-#define RTMP_SERVICES_VER_STR "rtmp-services plugin (libobs " OBS_VERSION ")"
+static const char *rtmp_services_log_str = "[rtmp-services plugin] ";
+static const char *rtmp_services_url = RTMP_SERVICES_URL;
```

Comment:
Creating static "global" variable is not a good solution, it should be avoided.

Reply:
How would you replace these instead? We require a solution for the oauth tokens as well to ensure they are properly type-checked.

Reply:
Use macros like it is actually done.

I don't why you are complicating yourself by adding those global variables.

Reply:
Because macros cannot be debugged, are not type-safe, can lead to strange side-effects, cannot be namespaced, and thus are "considered evil". Using enums or constant types should be preferred for constant values.

Reply:
> Creating static "global" variables is not a good solution, it should be avoided.

Should've asked directly: What's the negative effect of using a `const T` for this as in the change?

Reply:
Implicit rule of C/C++, IIRC. Use Macros for constants, unless you really need a `const T` for some reason.

Also do we need to always store those variables in a `const T` when they only used a few times and a macro avoids that.